core/domain/image_validation_services.py

```diff
@@ -38,14 +39,19 @@ def validate_image_and_filename(raw_image, filename):
             validation checks.
     """
     hundred_kb_in_bytes = 100 * 1024
-
     if not raw_image:
         raise utils.ValidationError('No image supplied')
     if len(raw_image) > hundred_kb_in_bytes:
         raise utils.ValidationError(
             'Image exceeds file size limit of 100 KB.')
     allowed_formats = ', '.join(
         list(feconf.ACCEPTED_IMAGE_FORMATS_AND_EXTENSIONS.keys()))
+    # TODO(#14204): Refactor this to use the normalized_request files.
+    raw_image = (
+        base64.decodebytes(raw_image.encode('utf-8'))
+        if utils.is_base64_encoded(raw_image)
+        else raw_image
+    )
```

Comment:
> PR #14063 adds schema validation for SuggestionHandler and introduces new approach to save multiple images.Since images are base64 encoded in some parts of code, so it's necessary that UpdateQuestionSuggestionHandler must also follow the same approach to handle images.

If "it's necessary" then why are we doing it in a separate PR? 

> Refactor this to use the normalized_request files.

What's "normalized_request files." here?

Reply:
```suggestion
    # TODO(#14204): Refactor this to use the normalized_request files.
    if utils.is_base64_encoded(raw_image):
        raw_image = (
            base64.decodebytes(raw_image.encode('utf-8'))
```

Reply:
Thank you for your valuable suggestions and here's what I think.

> If "it's necessary" then why are we doing it in a separate PR?

By the time I implemented this approach in `SuggestionHandler`, the `UpdateQuestionSuggestionHandler` was already assigned to someone.The following must be implemented:-

- Add schema in `UpdateQuestionSuggestionHandler` schema

```js
'files': {
                'schema': {
                    'type': 'object_dict',
                    'validation_method': (
                        domain_objects_validator.validate_suggestion_images
                    )
                },
                'default_value': None
            }
} 
```
-  and probably remove [this](https://github.com/chiragbaid7/oppia/blob/d86de726869fe630a503be3cb145eaee78f28579/core/controllers/suggestion.py#L570).

> What's "normalized_request files." here?

We are accessing `files` dict from `normalized_payload` to process and save images on disks. So I think it should either be renamed as normalized_payload or payload.